### PR TITLE
Update a-shafinal.md

### PR DESCRIPTION
**Pull request by RatinCN**
`A_SHAFinal` follows `A_SHAUpdate`, not `MD5Update`

### CODE REVIEW

File: desktop-src/SecCrypto/a-shafinal.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ api_location:
 
 # A\_SHAFinal function
 
-Computes the final hash of the data entered by the MD5Update function.
+Computes the final hash of the data entered by the A_SHAUpdate function.
```
Comment (alvinashcraft) [inline desktop-src/SecCrypto/a-shafinal.md:20]: Thanks for this update. What do you think about mentioning that the data digest should first be initialized with a call to the `A_SHAInit` function?

Comment (RatinCN) [reply desktop-src/SecCrypto/a-shafinal.md:20]: IMO, we needn't mention `A_SHAInit` here, and there are some suggestions may helpful:
1. `A_SHAInit` and `A_SHAUpdate` mention each other, `A_SHAUpdate` and `A_SHAFinal` mention each other, the relationship amount them could be clear
2. Link all of them in `See Also` section, after all, `XxxInit` -> `XxxUpdate` -> `XxxFinal` is standard flow for MD4, MD5, SHA, ...

I can contribute them in a separate PR if you think it's can be helpful

Comment (alvinashcraft) [reply desktop-src/SecCrypto/a-shafinal.md:20]: That sounds good. I'll approve and merge this PR, and if you'd like to submit another PR for the above changes, that would be great. Thanks!

**Review by alvinashcraft — APPROVED** [review]
These changes look good. Thanks for your contribution!

**Pull request merged**